### DISCUSSION TITLE
The Debug Graphs Visualizer

### DISCUSSION THREAD

**Discussion by SiliconVillager** [KhronosGroup/Vulkan-Samples#300] [Q&A] [answered]
Is the debug graph visualizer (https://github.com/KhronosGroup/Vulkan-Samples/blob/master/docs/debug_graphs.md) still available somewhere? I wasn't able to find it myself :(

**Comment by TomAtkinsonArm** [comment]
This is an extremely late reply!!

**Comment by TomAtkinsonArm** [comment, marked as answer]
Although the graphs are not available in their full form and I have no chance to finish this. I aim to add https://github.com/wolfpld/tracy support to help samples illustrate their use cases easier
